native/src/routes/LoadingErrorHandler.tsx

```diff
@@ -13,9 +14,10 @@ const LOADING_TIMEOUT = 800
 
 type LoadingErrorHandlerProps = {
   children?: ReactNode
-  error: Error | null
+  error: Error | ErrorCode | null
```

Comment:
Hmh, does this make sense? Why do we even have this ErrorCode class instead of creating own custom error class?

Reply:
In my opinion it does but not 100% sure. Imo handling of just an error code enum is so much more easier then creating a new error and initializing it for every error. I just figured since we don't ever use the additional information our error classes provide but instead we always just did `fromError(error)` which returns an error code we might as well use just the error code directly.

Reply:
@SteffiStuffel can you just open a ticket yourself please? Looking forward to see your proposal :)